Spark/metrics.csv

```diff
@@ -0,0 +1,207 @@
+spark.executor.total_shuffle_read.median,gauge,,byte,,The median total number of bytes read during a shuffle in the application's executors,0,spark,exe tot shfl rd med
+spark.executor.total_shuffle_read.95percentile,gauge,,byte,,The 95th percentile total number of bytes read during a shuffle in the application's executors,0,spark,exe tot shfl rd 95
+spark.executor.total_shuffle_read.count,rate,,byte,,The number of times the total number of bytes read during a shuffle in the application's executors was sampled,0,spark,exe tot shfl rd cnt
+spark.executor.total_shuffle_write.max,gauge,,byte,,The max total number of bytes written during a shuffle in the application's executors,0,spark,exe tot shfl wrt mx
+spark.executor.total_shuffle_write.avg,gauge,,byte,,The average total number of bytes written during a shuffle in the application's executors,0,spark,exe tot shfl wrt avg
+spark.executor.total_shuffle_write.median,gauge,,byte,,The median total number of bytes written during a shuffle in the application's executors,0,spark,exe tot shfl wrt med
+spark.executor.total_shuffle_write.95percentile,gauge,,byte,,The 95th percentile total number of bytes written during a shuffle in the application's executors,0,spark,exe tot shfl wrt 95
+spark.executor.total_shuffle_write.count,rate,,byte,,The number of times the total number of bytes written during a shuffle in the application's executors was sampled,0,spark,exe tot shfl wrt cnt
+spark.executor.max_memory.max,gauge,,byte,,The max maximum memory used in the application's executors,0,spark,exe max mem mx
+spark.executor.max_memory.avg,gauge,,byte,,The average maximum memory used in the application's executors,0,spark,exe max mem avg
+spark.executor.max_memory.median,gauge,,byte,,The median maximum memory used in the application's executors,0,spark,exe max mem med
+spark.executor.max_memory.95percentile,gauge,,byte,,The 95th percentile maximum memory used in the application's executors,0,spark,exe max mem 95
+spark.executor.max_memory.count,rate,,byte,,The number of times the maximum memory used in the application's executors was sampled,0,spark,exe max mem cnt
+spark.rdd.count,rate,,,,The number of persisted RDDs in the Spark application,0,spark,rdd cnt
+spark.rdd.num_partitions.max,gauge,,,,The max number of partitions in the application's persisted RDDs,0,spark,rdd num part mx
+spark.rdd.num_partitions.avg,gauge,,,,The average number of partitions in the application's persisted RDDs,0,spark,rdd num part avg
+spark.rdd.num_partitions.median,gauge,,,,The median number of partitions in the application's persisted RDDs,0,spark,rdd num part med
+spark.rdd.num_partitions.95percentile,gauge,,,,The 95th percentile number of partitions in the application's persisted RDDs,0,spark,rdd num part 95
+spark.rdd.num_partitions.count,rate,,,,The number of times the number of partitions in the application's persisted RDDs was sampled,0,spark,rdd num part cnt

```

Comment:
I'd change "the number of partitions in the application's persisted RDDs" to "the number of persisted RDD partitions in the application"